socorro/external/statsd/statsd_rule_benchmark.py

```diff
@@ -89,13 +88,8 @@ class CountStackWalkerTimeoutKills(CountAnythingRuleBase):
     )
 
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        # override me to check any condition within a raw, processed crash
-        # or even the state of the processor itself from the proc_meta
-        return reduce(
-            lambda x, y: x or "SIGKILL" in y,
-            proc_meta.processor_notes,
-            False
-        )
+        # Only run the action if "SIGKILL" is in the processor notes
+        return any(['SIGKILL' in note for note in proc_meta.processor_notes])
```

Comment:
Python 3 doesn't have `reduce`. I figured I'd rewrite this because the way it's written doesn't lend itself to sussing out what it's trying to do. I also ditched the original comment since it seems generic and unhelpful.